CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- New React Style - Card Kit ([#496][] @jasperfurniss)
```

Comment:
Can you add a more detailed description for Ninjas?

Reply:
Also move to the Add under `[Unreleased]`? :)